doc/eventhandler/index.rst

```diff
@@ -339,7 +339,16 @@ This can be used to e.g. automatically enroll a token for the user if the
 user has no tokens left (token_number == 0) of to notify the administrator if
 the user has to many tokens assigned.
 
+**counter**
 
+The counter condition can compare the value of any arbitrary event counter against a fixed
+value. Valid compares are:
+
+    myCounter == 1000
+    myCounter > 1000
+    myCounter < 1000
+
+"myCounter" being any event counter set with the :ref:`counterhandler`.
```

Comment:
Could You add some documentation about setting the counter value to `0` if the counter does not (yet) exists?

Reply:
Done in 4685a9f0